docs/syntax_guide.rst

```diff
@@ -125,7 +130,7 @@ See the :ref:`objects_and_classes` for details.
 ===================  ==============  ================================================
    **Property**       **Default**                    **Meaning**
 -------------------  --------------  ------------------------------------------------
- position [1]_        (0, 0)         position in global coordinates
+ position [1]_        (0, 0, 0)      position in global coordinates
  viewDistance          50            distance for the ‘can see’ operator
```

Comment:
Looks like the property is still called `visibleDistance`.

Also should add here `viewRayCount`, `viewRayDensity`, etc.